vispy/visuals/graphs/layouts/random.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""
+Random Graph Layout
+====================
+
+This layout positions the nodes at random
+"""
+
+import numpy as np
+
+from ..util import straight_line_vertices
+
+
+def random(adjacency_mat, directed=False):
+    if adjacency_mat.shape[0] != adjacency_mat.shape[1]:
+        raise ValueError("Adjacency matrix should be square.")
```

Comment:
This check is already done in `straight_line_vertices` so you don't need it here, right?